pkgs/development/libraries/openssl/default.nix

```diff
@@ -6,14 +6,9 @@
 # Used to avoid cross compiling perl, for example, in darwin bootstrap tools.
 # This will cause c_rehash to refer to perl via the environment, but otherwise
 # will produce a perfectly functional openssl binary and library.
-, withPerl ? stdenv.hostPlatform == stdenv.buildPlatform
+, withPerl ? true
```

Comment:
Note: I think it's confusing to flip `withPerl` based on whether you're cross-compiling or not. It seems clearer to just default to `true`.

Reply:
Yeah, but then you need to take care to not break things. #108170 #82924

Reply:
Hmm okay, thanks for looking those up. I just put it back.